### ISSUE TITLE
Unsupported JNI version detected

### ISSUE THREAD

**Issue by TheChocolateOre** [LWJGL/lwjgl3#814]
### Version

3.3.1

### Platform

Windows x64

### JDK

Oracle OpenJDK 19

### Module

LWJGL core

### Bug description

I'm getting a warning when running a LWJGL3-using app with Java 19 **previews enabled** (does not happen on 18). There is no crash, and business logic remains intact, still the warning might be of concern.

Of course, Java 19 was just released, so it's expected to encounter a few warnings. Nevertheless, I will give in to my year-long temptation and bring up [Project Panama](https://openjdk.org/projects/panama/). Java 19 shipped with [JEP 424](https://openjdk.org/jeps/424), so my question here is: 

_Are there any plans to migrate away from JNI and Buffers, eventually?_ 

I suspect the answer is yes, when _Foreign Function & Memory API_ makes it out of preview into a permanent feature. Will such a reform call for LWJGL **4**?

### Stacktrace or crash log output

```shell
[LWJGL] [ThreadLocalUtil] Unsupported JNI version detected, this may result in a crash. Please inform LWJGL developers.
```

**Comment by octylFractal** [comment]
See #813 about Project Panama.

**Comment by octylFractal** [comment]
And the JNI issue is fixed by #799, which is part of the 3.3.2 snapshot builds.
